tests/integration/targets/s3_bucket/roles/s3_bucket/tasks/ownership_controls.yml

```diff
@@ -0,0 +1,105 @@
+---
+- module_defaults:
+    group/aws:
+      aws_access_key: "{{ aws_access_key }}"
+      aws_secret_key: "{{ aws_secret_key }}"
+      security_token: "{{ security_token | default(omit) }}"
+      region: "{{ aws_region }}"
+  block:
+
```

Comment:
Please create and use a `local_bucket_name` fact like here: https://github.com/ansible-collections/amazon.aws/pull/345/files

Reply:
The PR looks good beside this. Nice patch @abikouo.